genericapi/handler.go

```diff
@@ -138,8 +138,22 @@ func (h *Handler) ServeCreateAlert(w http.ResponseWriter, r *http.Request) {
 		Status:    status,
 	}
 
+	var resp struct {
+		AlertID     int
+		ServiceID   string
+		IsDuplicate bool
+	}
+
 	err = retry.DoTemporaryError(func(int) error {
-		_, err = h.c.AlertStore.CreateOrUpdate(ctx, a)
+		createdAlert, err := h.c.AlertStore.CreateOrUpdate(ctx, a)
```

Comment:
We'll need to update `CreateOrUpdate` to return the `isNew` value as `createdAlert.Dedup` is the dedup hash, and will never be `nil`